reporters_db/data/reporters.json

```diff
@@ -18009,6 +18101,20 @@
             }
         }
     ],
+    "Reporter,": [
+        {
+            "cite_type": "federal",
+            "editions": {
+                "Reporter,": {
+                    "end": null,
+                    "start": null
+                }
+            },
+            "mlz_jurisdiction": [],
+            "name": "Reporter. (Continuation of Law and Equity Reporter and of American Law Times Reports, New Series.)",
+            "variations": {}
```

Comment:
this name ... where did you get it from

Reply:
Here: https://law.resource.org/pub/us/case/reporter/F.Cas/0001.f.cas/0001.f.cas.front.html

It also comes in the original volume, here: https://law.resource.org/pub/us/case/reporter/F.Cas/0001.f.cas.pdf at pdf's page 47

Reply:
great.